test/critest.bats

```diff
@@ -3,7 +3,7 @@
 load helpers
 
 function setup() {
-	if [[ -z $RUN_CRITEST ]]; then
+	if [[ -z "${RUN_CRITEST:-}" ]]; then
```

Comment:
Perhaps `if [ -v RUN_CRITEST ]` is easier / more readable?

Reply:
Ah, thanks ! I didn't know such a test existed in `bash`. Looking at it, I guess 4.0.2 is safe/old enough to assume as the least version. Updated tests accordingly.